analyzer/tests/functional/analyze_and_parse/test_files/cppcheck_warning.output

```diff
@@ -0,0 +1,40 @@
+NORMAL#CodeChecker log --output $LOGFILE$ --build "make cppcheck_multiple" --quiet
```

Comment:
please keep the output filename in sync with the actual file name: `cppcheck_multiple_warning.output` or `cppcheck_multiple.output`